system/include/emscripten/emscripten.h

```diff
@@ -244,6 +244,8 @@ char *emscripten_get_preloaded_image_data_from_FILE(FILE *file, int *w, int *h);
 #define EM_LOG_DEMANGLE 32
 #define EM_LOG_NO_PATHS 64
 #define EM_LOG_FUNC_PARAMS 128
+#define EM_LOG_DEBUG    256
+#define EM_LOG_INFO     512
```

Comment:
Seems like the level of the warning should not be bitfield since you can't have multiple levels at once, but thats really different change i guess.

Reply:
I've noticed this inconsistency, but I don't know what is the best way to solve it.
Ideally, emscripten_log() should have dedicated argument for message weight, but this will break API. It is also possible messing up with bit-fields, but this will unlikely improve anything.
So, indeed, it looks like a subject for dedicated patch.

Reply:
I think we can to it by allocating a few bits of the bitfield to be the integer log level.  

Se the code would look like:

```
var log_level = flag & LEVEL_MASK;
switch (log_level) {
   case LEVEL_DEBIG:
        ...etc...
```

Probably worth doing as a followup though.